kernel/README.md

```diff
@@ -1,13 +1,13 @@
 # Build Kata Containers Kernel
 
 This document explains the steps to build a compatible kernel with Kata
-Containers. To do this use build-kernel.sh, this script automates the
-process to build a kernel for Kata Containers. 
+Containers. To do this use build-kernel.sh, this script automates the process
+to build a kernel for Kata Containers. 
 
 ## Setup kernel source code
 
 ```bash
-./build-kernel.sh setup
+$ ./build-kernel.sh setup
 ```
 
 The script `./build-kernel.sh` tries to apply the patches from
```

Comment:
@jcvenegas Can you also describe that the setup command downloads the kernel source if not already present(I assume in the current directory?) and proceeds to applying the patches?

Reply:
Sure, I can do it in a next PR.